src/components/RightSidebar/Participants/ParticipantsTab.vue

```diff
@@ -20,33 +20,36 @@
 -->
 
 <template>
-	<div>
+	<div class="h-100">
 		<SearchBox v-if="canSearch"
 			:value.sync="searchText"
 			:is-focused.sync="isFocused"
 			:placeholder-text="searchBoxPlaceholder"
 			@input="handleInput"
 			@abort-search="abortSearch" />
 
-		<ParticipantsList v-if="!isSearching"
-			:items="participants"
-			:loading="!participantsInitialised" />
-
-		<template v-else>
-			<NcAppNavigationCaption v-if="canAdd" :title="t('spreed', 'Participants')" />
-
-			<ParticipantsList v-if="filteredParticipants.length"
-				:items="filteredParticipants"
-				:loading="!participantsInitialised" />
-			<Hint v-else :hint="t('spreed', 'No search results')" />
-
-			<ParticipantsSearchResults v-if="canAdd"
-				:search-results="searchResults"
-				:contacts-loading="contactsLoading"
-				:no-results="noResults"
-				:search-text="searchText"
-				@click="addParticipants" />
-		</template>
+		<div class="h-crop">
+			<ParticipantsListVirtual v-if="!isSearching"
+				:participants="participants"
+				:loading="!participantsInitialised"
+				class="h-100" />
+
+			<template v-else>
+				<NcAppNavigationCaption v-if="canAdd" :title="t('spreed', 'Participants')" />
+
+				<ParticipantsList v-if="filteredParticipants.length"
+					:items="filteredParticipants"
+					:loading="!participantsInitialised" />
+				<Hint v-else :hint="t('spreed', 'No search results')" />
```

Comment:
Cannot we use virtual scrolling also for search results here?

Reply:
Won't it be the same problem, as in LeftSidebar? I meant that we use here different components with unknown amount of elements, like filtered conversations + open conversations + e.t.c

Reply:
Yep, you are right. I missed `ParticipantsSearchResults` below